docs/1-Getting-Started/2-Starting-new-project.md

```diff
@@ -18,164 +12,164 @@ Also, make sure you're running the latest version of **kool**. Run the following
+Create a **docker-compose.yml** file in the root of your working directory, and then copy/paste one of the YAML templates provided below into **docker-compose.yml**.
 
 ```bash
-Preset laravel is initializing!
-
-? What app service do you want to use [Use arrows to move, type to filter]
-  PHP 7.4
-> PHP 8.0
-
-? What database service do you want to use [Use arrows to move, type to filter]
-> MySQL 8.0
-  MySQL 5.7
-  PostgreSQL 13.0
-  none
-
-? What cache service do you want to use [Use arrows to move, type to filter]
-> Redis 6.0
-  Memcached 1.6
-  none
-
-? What javascript package manager do you want to use [Use arrows to move, type to filter]
-  npm
-> yarn
-
-? What composer version do you want to use [Use arrows to move, type to filter]
-  1.x
-> 2.x
-
-$ Preset laravel initialized!
+$ touch docker-compose.yml
 ```
 
-Now, move into your new project.
+Here's a simple, generic setup for **docker-compose.yml**:
 
-```bash
-$ cd my-project
+```yaml
+version: "3.8"  # optional since v1.27.0
+services:
+  app:
+    image: kooldev/php:8.0-nginx
+    ports:
+      - "80:80"
+    volumes:
+      - .:/app:delegated
 ```
 
-The `kool preset` command auto-generated the following configuration files and added them to your project, which you can modify and extend.
+Here's a more extensible, **Kool-optimized** setup for **docker-compose.yml** into which you can easily add additional services (i.e. database, cache, etc):
 
-```diff
-+.dockerignore
-+docker-compose.yml
-+kool.yml
+```yaml
+version: "3.8"  # optional since v1.27.0
+services:
+  app:
+    image: kooldev/php:8.0-nginx
+    ports:
+      - ${KOOL_APP_PORT:-80}:80
+    environment:
+      ASUSER: ${KOOL_ASUSER:-0}
+      UID: ${UID:-0}
+    volumes:
+      - .:/app:delegated
+    networks:
+      - kool_local
+      - kool_global
+networks:
+  kool_local: null
+  kool_global:
+    external: true
+    name: ${KOOL_GLOBAL_NETWORK:-kool_global}
 ```
 
-> Open the **docker-compose.yml** file to review the services and verify they match the choices you made earlier using the wizard.
-
-### 2. Update .env.example
-
-You need to update Laravel's **.env.example** file with some default values that match the services in your **docker-compose.yml** file.
+> As you can see, we're using a [Kool-optimized Docker image](https://github.com/kool-dev/docker-php) for **PHP 8** (using `php:8.0-fpm-alpine`as its base), which also includes an NGINX web server. We're mapping **localhost** to container port `80`.
 
-#### Database Services
+### 3. Hello World!
 
-MySQL 5.7 and 8.0
+Create a `/public` sub-directory, and an **index.php** file inside it, and then copy/paste the code provided below into **index.php**.
 
-```diff
--DB_HOST=127.0.0.1
-+DB_HOST=database
+```bash
+$ mkdir public # in order to match NGINX's default root /app/public
+$ touch public/index.php
```

Comment:
I was gonna suggest having `vi` instead of `touch` but I see that could bring confusion to people unfamiliar with `vi`. Could we do something else? Just thinking that beginner developers might learn first about `vim` than `touch`.

Reply:
@fabriciojs - used `echo "..." > public/index.php` for the Hello World code, but left `touch` (for now) to create `docker-compose.yml`.